### PR TITLE
Issue #101: Onboarding process of Jonathan Lopez

### LINKED ISSUE #101: Onboarding process of Jonathan Lopez
Jonathan Lopez onboarding process

Tasks:

- [ ] Update time descriptions.
- [ ] Incorporate new DevSecOps resources into the training.

Acceptance Criteria:

The training should equip the developer with the skills necessary to work within the AI LAB's framework.
The training should provide accurate time estimates for each module.

### PR DESCRIPTION
**Pull request by SonOfLope**
Throughout Jonathan Lopez's onboarding, it was noticed that there was room for enhancement in the availability of security training resources.

### Tasks:
- [x] Update time descriptions.
- [x]  Incorporate new DevSecOps/security resources into the training.
- [x] Incorporate GoC documentation resources into the training.
### Acceptance Criteria:
- The training should equip the developer with the skills necessary to work within the AI LAB's framework.
- The training should provide accurate time estimates for each module.
### Closes PR:
- Closes #101

### CODE REVIEW

**Comment by rngadam** [comment]
would also appreciate some devsecops links and review of GoC documentation such as:

* https://github.com/canada-ca/cloud-guardrails
* https://www.canada.ca/en/shared-services/corporate/gc-cloud-strategy-2024.html

**Review by rngadam — CHANGES_REQUESTED** [review]
add GoC documentation

**Comment by SonOfLope** [comment]
> would also appreciate some devsecops links and review of GoC documentation such as:
> 
> * https://github.com/canada-ca/cloud-guardrails
> * https://www.canada.ca/en/shared-services/corporate/gc-cloud-strategy-2024.html

regarding GC Cloud strategy, it was announced to be published in [march 2024](https://gcxgce.sharepoint.com/teams/1000233/SitePages/Cloud-Strategy-Update-2024.aspx) : 
image

**Comment by rngadam** [comment]
@SonOfLope I gave the link to the guardrails too and would want to see some googling to see what else we can find related to devops and GoC. Example:

https://catalogue.csps-efpc.gc.ca/product?catalog=DDN112&cm_locale=en

**Comment by SonOfLope** [comment]
> @SonOfLope I gave the link to the guardrails too and would want to see some googling to see what else we can find related to devops and GoC. Example:
> 
> https://catalogue.csps-efpc.gc.ca/product?catalog=DDN112&cm_locale=en

Yes I am on that! I will request a new review upon adding new links.

File: TRAINING.md [hunk 1]
```diff
@@ -71,12 +74,28 @@
...
   * [Github Actions](https://docs.github.com/en/actions) (~1 heure)
   * [Azure Command-Line Interface (CLI)
     documentation](https://learn.microsoft.com/en-ca/cli/azure/) (~20 minutes)
   * [DevOps
     Engineer](https://learn.microsoft.com/en-us/certifications/devops-engineer/)
     (~5 minutes)
+  * [DevSecOps
+    Guidelines](https://owasp.org/www-project-devsecops-guideline/latest/00a-Overview)
+    (~30 minutes)
+  * [GoC Devops
```
Comment (rngadam) [inline TRAINING.md:89]: L'école de la fonction publique du Canada n'est pas accessible au public en général; je pense qu'il vaut mieux mettre ce lien dans notre wiki interne.

**Review by rngadam — CHANGES_REQUESTED** [review]

File: TRAINING.md (hunk 1 above)
Comment (SonOfLope) [reply TRAINING.md:89]: That link was your suggestion. I will remove it.

Comment (SonOfLope) [reply TRAINING.md]: done

Comment (rngadam) [reply TRAINING.md]: > That link was your suggestion. I will remove it.

yup, sometimes I don't give the right specifics but the intent is the same (to have a better DevOps training process for incoming employees and external contributors whether internal or external wiki).

**Review by rngadam — APPROVED** [review]

File: TRAINING.md (hunk 1 above)
Comment (SonOfLope) [reply TRAINING.md]: link was added to internal wiki 👍

**Pull request merged**